administrator/language/en-GB/com_guidedtours.sys.ini

```
@@ -0,0 +1,379 @@
+; Joomla! Project
+; (C) 2023 Open Source Matters, Inc. <https://www.joomla.org>
+; GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+COM_GUIDEDTOURS="Guided Tours"
+COM_GUIDEDTOURS_ERROR_TOUR_NOT_FOUND="Tour not found!"
+COM_GUIDEDTOURS_GUIDEDTOUR_STEPS_PAGE_TITLE="Guided Tours - List of Steps"
+COM_GUIDEDTOURS_GUIDEDTOURS_PAGE_TITLE="Guided Tours - List of Tours"
+COM_GUIDEDTOURS_GUIDEDTOURS_TABLE_CAPTION="List of Tours"
+COM_GUIDEDTOURS_TOURS_VIEW_DEFAULT_TITLE="List All Guided Tours"
+COM_GUIDEDTOURS_TOURS_VIEW_DEFAULT_DESC="Shows a list of all guided tours."
+COM_GUIDEDTOURS_XML_DESCRIPTION="Component for managing Guided Tours functionality in Joomla."
+
+COM_GUIDEDTOURS_TOUR_GUIDEDTOURS_TITLE="How to create a guided tour?"
+COM_GUIDEDTOURS_TOUR_GUIDEDTOURS_DESCRIPTION="<p>This tour will show you how you can create a guided tour for the Joomla backend.</p>"
```

```suggestion
COM_GUIDEDTOURS_TOUR_GUIDEDTOURS_DESCRIPTION="<p>This tour will show you how you can create a guided tour for the Joomla backend.</p>"
```
It is better to not include markup `<p>`. In this case it does not affect display.

---

We don't seen it in the initial language files, agreed.